secure-api-gateway-ob-uk-rs-server/src/main/java/com/forgerock/sapi/gateway/ob/uk/rs/server/configuration/RsApplicationConfiguration.java

```diff
@@ -78,15 +82,12 @@ public MappingJackson2HttpMessageConverter mappingJacksonHttpMessageConverter(@Q
     public Jackson2ObjectMapperBuilderCustomizer objectMapperBuilderCustomizer() {
         return (jacksonObjectMapperBuilder) -> {
             jacksonObjectMapperBuilder.timeZone(TimeZone.getDefault());
-            jacksonObjectMapperBuilder.serializationInclusion(JsonInclude.Include.NON_NULL);
```

Comment:
we were configuring this previously but the value was getting overwritten by Include.ALWAYS on line 89 of the old file